curator/src/mail_agent/newsletter_script.ts

```diff
@@ -0,0 +1,31 @@
+import { curateAndGenerateNewsletter } from './newsletter_txt_format';
+import dotenv from 'dotenv';
+
+dotenv.config();
+
+var postmark = require("postmark");
+
+// Calls the curateAndGenerateNewsletter function with right parameters then sends the mail
+// TODO : Add all the dynamic part (newsletter parameters, email params)
+
+async function runNewsletter() {
+    const {markdown, html} = await curateAndGenerateNewsletter();
+    //console.log(markdown);  // uncomment this part and comment below to test without postmark
+
+    // Sending email :
+    var client = new postmark.ServerClient(process.env.POSTMARK_SERVER_API_KEY as string);
```

Comment:
**issue:** You shoudl test that the `process.env.POSTMARK_SERVER_API_KEY` environment is set before initializing the postmark client.

Reply:
You shouldn't use `var` too. Use `let`or even better with `const` instead.
Var define variable in global scope. You may overwrite others variable value. [MDN](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/var)

`let` creates variable as block scope. You won't have the issue with var [var](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/let) You may still shadow variable.

`const` has the same scope than `let` but it forbids the variable to be reassigned, it's a constant.